corehq/apps/userreports/static/userreports/js/builder_view_models.js

```diff
@@ -91,6 +91,10 @@ hqDefine('userreports/js/builder_view_models.js', function () {
             }
             return hqImport('userreports/js/constants.js').DEFAULT_CALCULATION_OPTIONS;
         });
+        // for default filters, the value to filter by
+        this.filterValue = ko.observable("");
+        // for default filters, the dynamic date operator to filter by
+        this.filterOperator = ko.observable("");
```

Comment:
I wonder if it might be worth breaking some subclasses out of `PropertyListItem` at this point. I can't recall why I didn't do that in the first place, but seems like things will get unmanageable if we keep adding these situational fields to the model. I don't think this blocks this PR, but would probably be a nice change to make before we add any further fields.